TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/Models/FactorDataPayload.swift

```diff
@@ -22,6 +22,7 @@ import Foundation
 protocol FactorDataPayload {
   var friendlyName: String { get }
   var type: FactorType { get }
+  var allowIphoneMigration: Bool { get }
```

Comment:
We should include the allowIphoneMigration only for `CreateFactorPayload` because we are not supporting updating the value for `allowIphoneMigration`